src/app/screens/connect/btcSelectAddressScreen/index.tsx

```diff
@@ -0,0 +1,292 @@
+import styled, { useTheme } from 'styled-components';
+import { getAppIconFromWebManifest } from './helper';
+
+const TitleContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'hidden',
+  marginLeft: 30,
+  marginRight: 30,
+  marginTop: 48,
+});
+
+const OuterContainer = styled(animated.div)({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: 16,
+  marginRight: 16,
+});
+
+const AddressBoxContainer = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  marginTop: props.theme.spacing(12),
+}));
+
+const AddressBox = styled.div((props) => ({
+  width: 328,
+  height: 66,
+  padding: props.theme.spacing(10),
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  backgroundColor: props.theme.colors.elevation6_800,
+  marginBottom: 1,
+  ':first-of-type': {
+    borderTopLeftRadius: props.theme.radius(2),
+    borderTopRightRadius: props.theme.radius(2),
+  },
+  ':last-of-type': {
+    borderBottomLeftRadius: props.theme.radius(2),
+    borderBottomRightRadius: props.theme.radius(2),
+  },
+}));
+
+const AddressContainer = styled.div({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const TopImage = styled.img({
+  maxHeight: 32,
+  maxWidth: 32,
+});
+
+const Title = styled.h1((props) => ({
+  ...props.theme.typography.body_bold_l,
+  color: props.theme.colors.white_0,
+  marginTop: 12,
+}));
+
+const DapURL = styled.h2((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_400,
+  marginTop: props.theme.spacing(2),
+  textAlign: 'center',
+}));
+
+const AddressImage = styled.img({
+  width: 24,
+  height: 24,
+  marginRight: 8,
+});
+
+const AddressTextTitle = styled.h2((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'center',
+}));
+
+const TruncatedAddress = styled.h3((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_0,
+  textAlign: 'right',
+}));
+
+const BnsName = styled.h3((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_0,
+  textAlign: 'right',
+}));
+
+const RequestMessage = styled.p((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'left',
+  wordWrap: 'break-word',
+  marginTop: props.theme.spacing(12),
+  marginBottom: props.theme.spacing(12),
+}));
+
+const PermissionsTitle = styled.h3((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'left',
+  marginTop: 24,
+}));
+
+const Permission = styled.div((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_0,
+  marginTop: 12,
+  display: 'flex',
+  alignContent: 'center',
+}));
+
+const PermissionIcon = styled.div({
+  marginRight: 4,
+});
```

Comment:
scale and alignment of the icon looks a bit off

actual:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/f940caf3-9b28-41af-ab2c-d6f0470cdb74)

figma:
![Uploading image.png…]()

Reply:
scale was fixed, but vertical alignment was not: 

![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/cacfbae6-e3ce-4507-8653-e00e31588347)

Reply:
and figma has weight: bold, which is missing

![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/0fe6bc1d-9c26-4908-9de8-46d844106143)